MODULE.bazel

```diff
@@ -8,10 +8,18 @@ module(
 
 bazel_dep(name = "bazel_skylib", version = "1.5.0")
 bazel_dep(name = "aspect_bazel_lib", version = "2.5.3")
+bazel_dep(name = "container_structure_test", version = "1.16.0")
+bazel_dep(name = "rules_oci", version = "1.7.4")
 
 bazel_lib_toolchains = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "toolchains")
-bazel_lib_toolchains.tar()
 use_repo(bazel_lib_toolchains, "bsd_tar_toolchains")
+use_repo(bazel_lib_toolchains, "yq_darwin_amd64")
```

Comment:
are these conditional on what OS the user is running on? (is that a thing that is possible with bazel?)

Reply:
oh yes, these are conditional on where Bazel runs as we run yq in a repository rule, and repository rules are executed on the host, so we have to pick the right one for the host.